flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/ProgramTargetDescriptor.java

```diff
@@ -25,56 +25,28 @@
  */
 public class ProgramTargetDescriptor {
 
-	private final String clusterId;
-
 	private final String jobId;
 
-	private final String webInterfaceUrl;
-
-	public ProgramTargetDescriptor(String clusterId, String jobId, String webInterfaceUrl) {
-		this.clusterId = clusterId;
+	public ProgramTargetDescriptor(String jobId) {
 		this.jobId = jobId;
-		this.webInterfaceUrl = webInterfaceUrl;
-	}
-
-	public String getClusterId() {
-		return clusterId;
 	}
 
 	public String getJobId() {
 		return jobId;
 	}
 
-	public String getWebInterfaceUrl() {
-		return webInterfaceUrl;
-	}
-
 	@Override
 	public String toString() {
-		return String.format(
-			"Cluster ID: %s\n" +
-			"Job ID: %s\n" +
-			"Web interface: %s",
-			clusterId, jobId, webInterfaceUrl);
+		return "Job ID: %s\n" + jobId;
```

Comment:
I think you are intending to use String.format

Reply:
I agree with @zjffdu :)

Reply:
Before I had multiple fields in the `toString()` and then I reverted... 😅 Good catch!